src/app/admin/page.tsx

```diff
@@ -1,60 +1,251 @@
 "use client";
 import Navbar from "@/components/ui/navbar";
 import Footer from "@/components/ui/footer";
-import APLibrary from "@/components/landingPage/APLibrary";
-import { getUser } from "@/components/hooks/getUser";
+import { getUser } from "@/components/hooks/users";
 import { User } from "@/types/user";
 import { useState, useEffect } from "react";
 import { useRouter } from "next/navigation";
+import { useUserManagement } from "./useUserManagement";
+
+const apClasses = [
```

Comment:
nit- can you move this to an `apClasses.json` somewhere adjacent to this file for both re-use and conceptual isolation? This shouldn't be at the top of the file because incoming devs and maintainers shouldn't be reading it every time they open this file.